src/components/ProductCard/ProductCard.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+
+import cartSVG from '../../assets/dark_cart_icon.svg';
+import type Product from '../api/apiInterfaces.ts';
+
+import styles from './ProductCard.module.css';
+
+interface ProductCardAttributes {
+    product: Product;
+}
+
+const ProductCard: React.FC<ProductCardAttributes> = ({ product }) => {
+    const [currentCounter, setCounter] = useState(0);
+    const addToCart = () => {
```

Comment:
The logic for adding items to the cart should follow these steps:

1. When the user clicks the icon on the product, mark it (as it's currently done) and increment the counter in the header to indicate the addition to the cart.
2. When the user clicks again on the cart, remove the respective product from the cart and remove the marker indicating it's in the cart.

Reply:
@serhii-tsivinskyi-moc update the cart logic without using local storage